java-grupa8/Dealership/src/main/java/ro/siit/dealership/Dealership.java

```diff
@@ -0,0 +1,21 @@
+package ro.siit.dealership;
+
+/**
+ * @author x-RauL-x
+ */
+
+public class Dealership {
+    /**
+     *
+     * @param args
+     */
+    public static void main(String[] args) {
+     Customer [] customers = new Customer[1];
```

Comment:
The relationships between Dealership and other classes should be in the class model, not a static main mathod.